### PR TITLE
Package updates for Node 12.x

### PR DESCRIPTION
**Pull request by rajeevbbqq**

### CODE REVIEW

File: packages/nodejs/src/ServiceClient.js [hunk 1]
```diff
@@ -96,7 +95,7 @@ class ServiceClient extends BaseServiceClient {
   _getGrpcChannelCredentials(serviceEndpoint) {
     if(serviceEndpoint.protocol === 'https:') {
       logger.debug('Channel credential created for https', { tags: ['gRPC'] });
-      return grpc.credentials.createSsl();
+      return grpc.credentials.createSsl()
```
Comment (anandrgitnirman) [inline packages/nodejs/src/ServiceClient.js:98]: here we have removed the semicolon

File: packages/nodejs/src/ServiceClient.js [hunk 2]
```diff
@@ -37,8 +36,8 @@ class ServiceClient extends BaseServiceClient {
   }
 
   setConcurrencyTokenAndChannelId(token, channelId) {
-    this.concurrencyManager.token = token
-    this._paymentChannelManagementStrategy.channelId = channelId
+    this.concurrencyManager.token = token;
+    this._paymentChannelManagementStrategy.channelId = channelId;
```
Comment (anandrgitnirman) [inline packages/nodejs/src/ServiceClient.js:40]: here we have added semicolon , can you opine on why we need ; at some places and not other others ( its removed and its added as well )

File: packages/nodejs/package.json [hunk 3]
```diff
@@ -32,25 +32,25 @@
...
   "devDependencies": {
-    "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.3.4",
-    "@babel/node": "^7.2.2",
-    "@babel/preset-env": "^7.3.4",
-    "@babel/register": "^7.0.0",
-    "google-protobuf": "^3.8.0",
-    "rimraf": "^2.6.3",
-    "verdaccio": "^4.8.1"
+    "@babel/cli": "^7.13.10",
```
Comment (anandrgitnirman) [inline packages/nodejs/package.json:47]: are all these recommended upgrades ?

File: packages/nodejs/src/ServiceClient.js (hunk 2 above)
Comment (rajeevbbqq) [reply packages/nodejs/src/ServiceClient.js:51]: it was an auto linting issue and fixed in latest commit

File: packages/nodejs/package.json (hunk 3 above)
Comment (rajeevbbqq) [reply packages/nodejs/package.json:47]: Added @babel/cli as a development dependency that is required to debug SDK locally or we need to explicitly add.

**Review by rajeevbbqq — COMMENTED** [review]
Please review.

**Review by Vivek205 — APPROVED** [review]

File: packages/nodejs/src/ServiceClient.js [hunk 4]
```diff
@@ -94,12 +104,12 @@ class ServiceClient extends BaseServiceClient {
   }
 
   _getGrpcChannelCredentials(serviceEndpoint) {
-    if(serviceEndpoint.protocol === 'https:') {
+    if (serviceEndpoint.protocol === 'https:') {
```
Comment (anandrgitnirman) [inline packages/nodejs/src/ServiceClient.js:107]: lets stndrdize the indentaion too , ths should not come up in diff

Comment (rajeevbbqq) [reply packages/nodejs/src/ServiceClient.js]: Resolved

**Pull request merged**